### ISSUE TITLE
impl Debug the same as to_string

### ISSUE THREAD

**Issue by FauxFaux** [akubera/bigdecimal-rs#35]
The current `Debug` implementation isn't super useful for debugging:

```
  left: `BigDecimal { int_val: BigInt { sign: Plus, data: BigUint { data: [4] } }, scale: 0 }`,
 right: `BigDecimal { int_val: BigInt { sign: Plus, data: BigUint { data: [1431655765, 1431655765, 1431655765, 940541189, 954603096, 3658156244, 4225298282, 742496771, 3561904397, 2399962859, 1560] } }, scale: 100 }`', lib/src/eval_tests.rs:13:5
```

I would much rather have the `to_string()` result as the `Debug` implementation? I assume the above is meaningless to the author, too. :)

```
  left: `"4"`,
 right: `"0.3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333"`', lib/src/eval_tests.rs:16:5
```

**Comment by kestred** [comment]
Might be nice to do something like `BigDecimal("0.33333333")` instead of just a pure string view

**Comment by akubera** [comment]
Implemented in #41; Looking at the raw numbers of the BigInt underneath is helpful sometimes when working with say implementing multiplication, but this is obviously the better option for people using the library.

Expect a new version soon.
